composer.json

```diff
@@ -7,7 +7,8 @@
     "require": {
         "php": ">=5.5.9",
         "laravel/framework": "5.1.*",
-        "laravel/socialite": "^3.0@dev"
+        "laravel/socialite": "^3.0@dev",
+        "abraham/twitteroauth": "^0.6.4"
```

Comment:
It appears there is in master (thujohn/twitter), but wasn't in the complete-refactor branch.

Master was doing [quite a bit](https://github.com/mattstauffer/confomo/blob/master/app/Twitter/SyncProfile.php) more work than what my proposed changes are. We're not (currently) syncing Twitter profiles, only the avatars. Thoughts?